scripts/map.js

```diff
@@ -98,10 +98,10 @@ var loadZones = function (geojson) {
     },
   }).addTo(map)
 
-  // Turn on federal/state land by default
-  $('input[name="Overlay"][value="federal"]').prop('checked', true)
-  $('input[name="Overlay"][value="state"]').prop('checked', true)
-  $('input[name="Overlay"][value="DHHL"]').prop('checked', true)
+  // Turn off federal/state land by default
+  $('input[name="Overlay"][value="federal"]').prop('checked', false)
+  $('input[name="Overlay"][value="state"]').prop('checked', false)
+  $('input[name="Overlay"][value="DHHL"]').prop('checked', false)
```

Comment:
```suggestion
```

Perhaps it would just be better to just delete these lines

Reply:
are we not ever turning these back on in the future?

Reply:
They are unchecked by default

Reply:
IMO it's better to just delete them since we can add these lines back anyway if we are going to toggle them on start - leaving them in while basically just preserving the default behaviour is just confusing